libtiledbsoma/src/CMakeLists.txt

```diff
@@ -197,6 +198,13 @@ install(FILES
   DESTINATION "include/tiledbsoma"
 )
 
+install(FILES
```

Comment:
Why are we installing the threadpool headers?

Reply:
Because I'm using the threadpool